assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+            "and": []
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "power_supply=yes"
+                ]
+              },
+              "then": {
+                "en": "This place has a power supply"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "power_supply=no"
+                ]
+              },
+              "then": {
+                "en": "This place does not have power supply"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "This place charges {charge}"
+          },
+          "question": {
+            "en": "How much does this place charge?"
+          },
+          "mappings": [],
+          "freeform": {
+            "key": "charge"
+          },
+          "condition": {
+            "and": [
+              "fee=yes"
+            ]
+          }
+        },
+        {
+          "render": {
+            "en": "This place is called {name}"
+          },
+          "question": {
+            "en": "What is this place called?"
+          },
+          "freeform": {
+            "key": "name"
+          },
+          "condition": {
+            "and": [
+              "name="
+            ]
+          }
+        },
+        {
+          "render": {
+            "en": "Sanitary dump station"
+          },
+          "question": {
+            "en": "Does this place have a sanitary dump station?"
+          },
+          "freeform": {
+            "key": "",
+            "addExtraTags": []
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station=yes"
+                ]
+              },
+              "then": {
+                "en": "This place has a sanitary dump station"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station=no"
+                ]
+              },
+              "then": {
+                "en": "This place does not have a sanitary dump station"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "This place is operated by {operator}"
+          },
+          "question": {
+            "en": "Who operates this place?"
+          },
+          "freeform": {
+            "key": "operator"
+          }
+        },
+        {
+          "render": {
+            "en": "{capacity} campers can use this place at the same time"
+          },
+          "question": {
+            "en": "How many campers can stay here? (skip if there is no obvious number of spaces or allowed vehicles)"
+          },
+          "freeform": {
+            "key": "capacity",
+            "type": "pnat"
+          }
+        },
+        {
+          "render": {
+            "en": "internet access"
```

Comment:
This is a weird rendering - it doesn't contain any substitutions... Maybe remove it, together with the freeform block? If you want to allow freeform text input, then at least use `Internet is accessible via {internet_access}` or something, e.g. to make sense for values like `wlan`, `wifi`,...

Reply:
right, removing those and trying to render wlan and wifi as "yes" without it being an answer option